recipe/meta.yaml

```diff
@@ -12,18 +12,19 @@ source:
 build:
    number: 0
    features:
-     - vc9      [win and py27]
-     - vc10     [win and py34]
-     - vc14     [win and py35]
+     - vc9      # [win and py27]
+     - vc10     # [win and py34]
+     - vc14     # [win and py35]
 
 requirements:
   build:
-    - gmp      [not win]
+    - gmp      # [not win]
+    # Python is listed as a build dependency purely to enumerate the different
+    # vc versions (i.e. we need one build per vc version).
+    - python   # [win]
 
   run:
-    # python is a Windows run dependency in order to get the correct vc feature
-    - python   [win]
-    - gmp      [not win]
+    - gmp      # [not win]
```

Comment:
It won't transparently however. You will likely have to patch the Makefile
and potentially the source (or better yet submit upstream).

On Thursday, March 24, 2016, jakirkham <notifications@github.com> wrote:

> In recipe/meta.yaml
> <https://github.com/conda-forge/glpk-feedstock/pull/1#discussion_r57410264>
> :
>
> >
> >    run:
> > -    # python is a Windows run dependency in order to get the correct vc feature
> > -    - python   [win]
> > -    - gmp      [not win]
> > +    - gmp      # [not win]
>
> It might link with MPIR, which is in the pipe.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/glpk-feedstock/pull/1/files/73fabc90f385759251d1f047374e3e98f133697c#r57410264>
>